### DISCUSSION TITLE
Image size for inserted images

### DISCUSSION THREAD

**Discussion by RomanLF** [quarto-dev/quarto-cli#2526] [Q&A] [answered]
Hello, is there a way to control the size of inserted images such as :

```
![image](some_image.PNG){#fig-an_image}{#width = 50%}
```
?

I know that it is possible on markdown with src/html synthax but I would like to know the best practice for quarto.

Best regards,

Roman

**Comment by dragonstyle** [comment, marked as answer]
Using `height` and `width` attributes are the preferred approach - these are handled specially by pandoc:


> The width and height attributes on images are treated specially. When used without a unit, the unit is assumed to be pixels. However, any of the following unit identifiers can be used: px, cm, mm, in, inch and %. There must not be any spaces between the number and the unit. For example:

> Dimensions may be converted to a form that is compatible with the output format (for example, dimensions given in pixels will be converted to inches when converting HTML to LaTeX). Conversion between pixels and physical measurements is affected by the [--dpi](https://pandoc.org/MANUAL.html#option--dpi) option (by default, 96 dpi is assumed, unless the image itself contains dpi information).

so 

```markdown
![Image Caption](image_file.png){#fig-foo width="50%"}
```

**Reply by RomanLF** [reply]
Thank you a lot for your explanation. It works well.
